benchmarks/csharp/Program.cs

```diff
@@ -18,29 +18,29 @@ private enum ChosenAction { GET_NON_EXISTING, GET_EXISTING, SET };
 
     public class CommandLineOptions
     {
-        [Option('r', "resultsFile", Required = true, HelpText = "Set the file to which the JSON results are written.")]
-        public string resultsFile { get; set; } = "";
+        [Option('r', "resultsFile", Required = false, HelpText = "Set the file to which the JSON results are written.")]
+        public string resultsFile { get; set; } = "../results/csharp-results.json";
 
-        [Option('d', "dataSize", Required = true, HelpText = "The size of the sent data in bytes.")]
-        public int dataSize { get; set; } = -1;
+        [Option('d', "dataSize", Required = false, HelpText = "The size of the sent data in bytes.")]
+        public int dataSize { get; set; } = 100;
 
-        [Option('c', "concurrentTasks", Required = true, HelpText = "The number of concurrent operations to perform.")]
-        public IEnumerable<int> concurrentTasks { get; set; } = Enumerable.Empty<int>();
+        [Option('c', "concurrentTasks", Required = false, HelpText = "The number of concurrent operations to perform.", Default = new[] { 1, 10, 100, 1000 })]

```

Comment:
why some of the defaults are being set like that and others like dataSize differently?

Reply:
IMO direct assignment is more readable, but it's not possible for collections.